{{cookiecutter.repo_name}}/{{cookiecutter.readme}}

```diff
@@ -4,17 +4,22 @@
 [![PyPI Version](https://img.shields.io/pypi/v/{{ cookiecutter.distribution_name }})](https://pypi.org/project/{{ cookiecutter.distribution_name }}/)
 ![License](https://img.shields.io/pypi/l/{{ cookiecutter.distribution_name }}?color=blue)
 
-{{ cookiecutter.description }}
+{{ cookiecutter.description }}.
```

Comment:
What happens when someone includes a period in the `description` field when answering the interactive questions? Won't that (unexpectedly) result in two periods? If so, perhaps the "principle of least astonishment" approach is to let the user handle the period?

Reply:
Aha OK, I'll remove it. It was just nitpicking anyway! 😁